Configuration/Eras/python/Era_Run2_2016_pA_cff.py

```diff
@@ -2,6 +2,7 @@
 
 from Configuration.Eras.Era_Run2_2016_cff import Run2_2016
 from Configuration.Eras.Modifier_pA_2016_cff import pA_2016
+from Configuration.Eras.Modifier_strips_vfp30_2016_cff import strips_vfp30_2016
 
-Run2_2016_pA = cms.ModifierChain(Run2_2016, pA_2016)
+Run2_2016_pA = cms.ModifierChain(Run2_2016.copyAndExclude([strips_vfp30_2016]), pA_2016)
```

Comment:
I'm not sure if it matters here.
pA_2016 is used only in reco; was the DIGI step made with pA as well?
Perhaps it's safer to remove, anyways.

Reply:
as far as I can see in the only relval using the era `Run2_2016_pA` in release the era is used in all steps: (including digitization):
```
281.0  EPOS_PPb_8160GeV_MinimumBias+DIGIUP15_PPb+RECOUP15_PPb+HARVESTUP15_PPb [1]: cmsDriver.py ReggeGribovPartonMC_EposLHC_4080_4080GeV_pPb_cfi  --conditions auto:run2_mc_pa -n 10 --era Run2_2016_pA --eventcontent RAWSIM --relval 9000,100 -s GEN,SIM --datatier GEN-SIM --beamspot RealisticPPbBoost8TeV2016Collision 
                                           [2]: cmsDriver.py step2  --conditions auto:run2_mc_pa -s DIGI:pdigi_valid,L1,DIGI2RAW,HLT:PIon --datatier GEN-SIM-DIGI-RAW-HLTDEBUG -n 10 --era Run2_2016_pA --eventcontent FEVTDEBUGHLT
                                           [3]: cmsDriver.py step3  --runUnscheduled  --conditions auto:run2_mc_pa -s RAW2DIGI,L1Reco,RECO,ALCA:TkAlMinBias+TkAlMuonIsolatedPA+TkAlUpsilonMuMuPA+TkAlZMuMuPA,EI,VALIDATION,DQM --datatier AODSIM,DQMIO -n 10 --era Run2_2016_pA --eventcontent AODSIM,DQM
                                           [4]: cmsDriver.py step4  --filetype DQM --mc  --conditions auto:run2_mc_pa --era Run2_2016_pA -s HARVESTING:validationHarvestingNoHLT+dqmHarvestingFakeHLT -n 100
```